src/compute/src/render/join/mz_join_core.rs

```diff
@@ -419,13 +420,24 @@ where
 
         let temp = &mut self.temp;
 
+        let flush = |data: &mut Vec<_>, session: &mut Session<_, _, _>| {
+            let old_len = data.len();
+            // TODO: This consolidation is optional, and it may not be very
+            //       helpful. We might try harder to understand whether we
+            //       should do this work here, or downstream at consumers.
```

Comment:
I feel like this comment makes it seem like doing the consolidation here is not very important, when in fact we have just seen that it is. Should we remove or reword it?

Reply:
Updated, thanks.